iceoryx_posh/source/roudi/application/roudi_main.cpp

```diff
@@ -38,9 +39,7 @@ int main(int argc, char* argv[])
             })
             .value();
 
-    cmdLineParser.printParameters();
-
-    IceOryxRouDiApp roudi(cmdLineParser, roudiConfig);
+    IceOryxRouDiApp roudi(cmdLineArgs, roudiConfig);
```

Comment:
It would be nice to only have the `roudiConfig` here. Ideally such things like the LogLevel could also set by TOML but the cmd line would have a higher precedence. Maybe something for another PR.

Reply:
Yeah, good point. For me this is related to the RouDi refactorings for safety certificiation. This epic should be started with #230